main_test.go

```diff
@@ -26,15 +26,15 @@ func TestRunsSuite(t *testing.T) {
 	//	dns.SetManifestPath("testdata/my-custom-solver"),
 	//	dns.SetBinariesPath("_test/kubebuilder/bin"),
 	//)
-
 	solver := example.New("59351")
 	fixture := dns.NewFixture(solver,
 		dns.SetResolvedZone("example.com."),
 		dns.SetManifestPath("testdata/my-custom-solver"),
-		dns.SetBinariesPath("_test/kubebuilder/bin"),
 		dns.SetDNSServer("127.0.0.1:59351"),
 		dns.SetUseAuthoritative(false),
 	)
 
-	fixture.RunConformance(t)
```

Comment:
@roytev https://github.com/cert-manager/cert-manager/pull/4835 should fix this problem, could you add a comment reminding us to change this back once the fix is released in cert-manager?

Reply:
Sure done :)